assets/scripts/oppia-player-0.0.1.js

```diff
@@ -62,7 +62,7 @@
   var SECRET_LENGTH = 64;
   var secret = '';
   for (var i = 0; i < SECRET_LENGTH; i++) {
-    secret += String.fromCharCode(65 + Math.floor(Math.random() * 26));
+    secret += String.fromCharCode(65 + Math.floor(random.SystemRandom().random() * 26));
```

Comment:
Is this valid JavaScript? I think this is the call for randomness in Python. MDN says to use [`window.crypto.getRandomValues()`](https://developer.mozilla.org/en-US/docs/Web/API/Crypto/getRandomValues) (though I'm hardly a JS expert)

ditto elsewhere

Reply:
Done